README.md

```diff
@@ -92,14 +92,17 @@ You can use
 
 ## JAR 
 
-* Download the [JAR file](https://github.com/zeebe-io/zeebe-http-worker/releases) 
+* Build the JAR file 
+
+    `mvn clean build`
+
 * Execute the JAR via
 
     `java -jar target/zeebe-lambda-worker-{VERSION}.jar`
 
 ## Docker
 
-    `docker run camunda/zeebe-lambda-worker`
+    docker run camunda/zeebe-lambda-worker:SNAPSHOT
```

Comment:
Why do you use SNAPSHOT as default - I think this is not a best practice and I would not advise to do so in the readme

Reply:
@berndruecker normally you'd be absolutely right. unfortunately, the only image currently published to Docker Hub is the one with the `SNAPSHOT` tag so without explicitly linking to `SNAPSHOT` it would be confusing for folks looking to test drive the project (took me a couple of hours to catch up to this and I have Docker experience)